src/Data/List/Properties.agda

```diff
@@ -1294,6 +1294,14 @@ module _ {P : Pred A p} (P? : Decidable P) where
   ...  | true  = Product.map s≤s m≤n⇒m≤1+n ih
   ...  | false = Product.map m≤n⇒m≤1+n s≤s ih
 
+  partition-is-foldr : partition P? ≗ foldr (λ x → if does (P? x) then Product.map₁ (x ∷_)
+                                                                  else Product.map₂ (x ∷_))
+                                            ([] , [])
```

Comment:
I'd be tempted to insert some linebreaks:
```suggestion
  partition-is-foldr : partition P? ≗ foldr
    (λ x → if does (P? x) then Product.map₁ (x ∷_) else Product.map₂ (x ∷_))
    ([] , [])
```
here, but not necessarily in the `CHANGELOG`.